v3/src/components/app.scss

```diff
@@ -1,4 +1,4 @@
-#app {
+#codap-app-id {
```

Comment:
Is the `id` part of this ID value just to differentiate it from the similar class name? It's fine, I think, but it did raise a yellow flag for me. It seems overly self-descriptive, but maybe that's good. It certainly won't hurt anything.